roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -1,13 +1,13 @@
 ---
 - name: Create passthrough route for docker-registry
-  command: >
-    {{ openshift.common.client_binary }} create route passthrough
-    --service docker-registry
-    --config={{ openshift_hosted_kubeconfig }}
-    -n default
-  register: create_docker_registry_route
-  changed_when: "'already exists' not in create_docker_registry_route.stderr"
-  failed_when: "'already exists' not in create_docker_registry_route.stderr and create_docker_registry_route.rc != 0"
+  oc_route:
+    kubeconfig: "{{ openshift_hosted_kubeconfig }}"
```

Comment:
This *should* be using openshift_hosted_kubeconfig defined here: https://github.com/ashcrow/openshift-ansible/blob/d4dbf337bdc77fc73d9bcad599205c429cdb91b4/roles/openshift_hosted/tasks/main.yml#L8

That said, there shouldn't be a need to use a copy of the kubeconfig here, since the module will already make a copy of the kubeconfig.